sdk/hybridcompute/Azure.ResourceManager.HybridCompute/src/Generated/HybridComputeMachineData.cs

```diff
@@ -32,21 +37,135 @@ public HybridComputeMachineData(AzureLocation location) : base(location)
         public ManagedServiceIdentity Identity { get; set; }
+        /// <summary> Metadata pertaining to the geographic location of the resource. </summary>
+        public LocationData LocationData { get; set; }
+        /// <summary> Configurable properties that the user can set locally via the azcmagent config command, or remotely via ARM. </summary>
+        public AgentConfiguration AgentConfiguration { get; }
+        /// <summary> Statuses of dependent services that are reported back to ARM. </summary>
+        public HybridComputeServiceStatuses ServiceStatuses { get; set; }
+        /// <summary> The metadata of the cloud environment (Azure/GCP/AWS/OCI...). </summary>
+        internal CloudMetadata CloudMetadata { get; set; }
+        /// <summary> Specifies the cloud provider (Azure/AWS/GCP...). </summary>
+        public string CloudMetadataProvider
+        {
+            get => CloudMetadata is null ? default : CloudMetadata.Provider;
+        }
+
+        /// <summary> The info of the machine w.r.t Agent Upgrade. </summary>
+        public AgentUpgrade AgentUpgrade { get; set; }
+        /// <summary> Specifies the operating system settings for the hybrid machine. </summary>
+        public OSProfile OSProfile { get; set; }
+        /// <summary> The provisioning state, which only appears in the response. </summary>
+        public string ProvisioningState { get; }
+        /// <summary> The status of the hybrid machine agent. </summary>
+        public HybridComputeStatusType? Status { get; }
+        /// <summary> The time of the last status change. </summary>
+        public DateTimeOffset? LastStatusChange { get; }
+        /// <summary> Details about the error state. </summary>
+        public IReadOnlyList<ResponseError> ErrorDetails { get; }
+        /// <summary> The hybrid machine agent full version. </summary>
+        public string AgentVersion { get; }
+        /// <summary> Specifies the hybrid machine unique ID. </summary>
+        public string VmId { get; set; }
+        /// <summary> Specifies the hybrid machine display name. </summary>
+        public string DisplayName { get; }
+        /// <summary> Specifies the hybrid machine FQDN. </summary>
+        public string MachineFqdn { get; }
+        /// <summary> Public Key that the client provides to be used during initial resource onboarding. </summary>
+        public string ClientPublicKey { get; set; }
+        /// <summary> The Operating System running on the hybrid machine. </summary>
+        public string OSName { get; }
+        /// <summary> The version of Operating System running on the hybrid machine. </summary>
+        public string OSVersion { get; }
+        /// <summary> The type of Operating System (windows/linux). </summary>
+        public string OSType { get; set; }
+        /// <summary> Specifies the Arc Machine's unique SMBIOS ID. </summary>
+        public string VmUuid { get; }
```

Comment:
Can we change the type to [Guid](https://learn.microsoft.com/en-us/dotnet/api/system.guid?view=net-6.0)? See example [here](https://apiview.dev/Assemblies/Review/25a98e223bda4f39b4ef76bebffa2c23#d4cebe383870410cab2e14d0709f78bb).

Reply:
changed the type of VmUuid to Guid.